app/assets/javascripts/AdverseEvent.js

```diff
@@ -14,9 +16,9 @@ const [Number, String] = [
 const AdverseEventSchema = DataElementSchema({
   authorDatetime: DateTime,
   relevantPeriod: Interval,
-  severity: Code,
-  facilityLocation: Code,
-  type: Code,
+  severity: Any,
```

Comment:
is this the direction we wanted to go in? I missed half of the discussion about this, but it seems like moving from `Code` to `Any` makes it so we may run into equivalence issues if one has a "description" and the other doesn't (and these *should* be equivalent in that case if these are codes).

Reply:
Changing type Code (Our basetype Code.js) to Any is what causes the Any casting in Any.js to be used which will cast codes to cql.Code before being used by the execution engine. This results in the execution engine always using cql.Code instead of Object as code as it was when we were using basetype/Code.js.